### PR TITLE
Resizing with window borders alters zoom to preserve content aspect ratio

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: dc/core/Content.h [hunk 1]
```diff
@@ -97,6 +98,9 @@ class Content : public QObject
     /** Get the aspect ratio. */
     qreal getAspectRatio() const;
 
+    /** @return true if the content does not have a fixed aspect ratio. */
```
Comment (tribal-tec) [inline dc/core/Content.h]: ...does have...

File: dc/core/ContentWindowController.cpp [hunk 2]
```diff
@@ -51,6 +52,7 @@ const qreal MIN_VISIBLE_AREA_PX = 300.0;
 const qreal INSIDE_MARGIN = 0.05;
 const qreal LARGE_SIZE_SCALE = 0.75;
 const qreal WINDOW_CONTROLS_MARGIN_PX = 175.0;
+const qreal ONE_PRECENT = 0.01;
```
Comment (tribal-tec) [inline dc/core/ContentWindowController.cpp]: ONE_PERCENT

**Comment by tribal-tec** [comment]
+1 otherwise

**Comment by rdumusc** [comment]
thanks, fixed

**Pull request merged**